pkgs/tools/networking/openssh/default.nix

```diff
@@ -39,6 +39,7 @@ stdenv.mkDerivation rec {
   # properly when cross building.
   configureFlags = [
     "--localstatedir=/var"
+    "--with-pid-dir=/var/run"
```

Comment:
Yes, `/var/run` is a symlink to `/run` on NixOS, so we should probably use the latter value to begin with.